transaction-view/src/signature_meta.rs

```diff
@@ -6,10 +6,20 @@ use {
     solana_sdk::{packet::PACKET_DATA_SIZE, pubkey::Pubkey, signature::Signature},
 };
 
+// The packet has a maximum length of 1232 bytes.
+// Each signature must be paired with a unique static pubkey, so each
+// signature really requires 96 bytes. This means the maximum number of
+// signatures in a **valid** transaction packet is 12.
+// In our u16 encoding scheme, 12 would be encoded as a single byte.
+// Rather than using the u16 decoding, we can simply read the byte and
+// verify that the MSB is not set.
+const MAX_SIGNATURES_PER_PACKET: u8 =
```

Comment:
moved this and related accounts definition to outside the function, since now the type of `num_signatures` on the struct is dependent on this value being storable in a u8.

Reply:
Also nicely lets me use the consts in the testing code